### PR TITLE
doc.ddoc: Add dub to sidebar

### PR DESCRIPTION
**Pull request by schancel**
Dub should be under the tools section.

### CODE REVIEW

**Comment by schancel** [comment]
Should we add dfmt and dfix as well?

**Comment by DmitryOlshansky** [comment]
Looks good, and I think we should list dfmt and dfix too.

**Comment by schancel** [comment]
@DmitryOlshansky Not sure what address to put in for those tools.  They only seem to have github projects.

**Comment by DmitryOlshansky** [comment]
> @DmitryOlshansky Not sure what address to put in for those tools. They only seem to have github projects.

I jut looked and these  github projects have sensible README page so I should be fine to link them on github for now.

File: doc.ddoc [hunk 1]
```diff
@@ -71,6 +71,7 @@ $(DIVID cssmenu, $(UL
         dmd-windows.html, dmd $(NDASH) reference compiler,
         http://gdcproject.org, gdc $(NDASH) gcc-based compiler,
         http://wiki.dlang.org/LDC, ldc $(NDASH) LLVM-based compiler,
+        http://code.dlang.org, dub $(NDASH) D package manager,
```
Comment (MartinNowak) [inline doc.ddoc]: We already have a code.dlang.org link above, though "more libraries" should be changed to "DUB - The D package registry".
You might put a link to http://code.dlang.org/download in the tool section.

**Comment by schancel** [comment]
@DmitryOlshansky @MartinNowak How does this look?  Not sure why the error check is failing. :/

**Comment by MartinNowak** [comment]
thx

**Comment by MartinNowak** [comment]
follow-up #1061

**Pull request merged**